app/controllers/phone_verifications_controller.rb

```diff
@@ -27,17 +27,11 @@ def verify
       phone_number: session[:phone_number]
     )
     if @response.ok?
-      redirect
+      current_user.update_attribute(:confirmed, true) && (redirect_to success_phone_verifications_path)
```

Comment:
```suggestion
      redirect_to success_phone_verifications_path if current_user.update_attribute(:confirmed, true)
```

btw why `update_attribute` instead of just `update`?